include/z64.h

```diff
@@ -1516,15 +1511,15 @@ struct GlobalContext {
     /* 0x01CA0 */ ActorContext actorCtx;
     /* 0x01F24 */ CutsceneContext csCtx;
     /* 0x01F74 */ CutsceneEntry* cutsceneList;
-    /* 0x01F78 */ GlobalContext1F78 unk1F78[16];
+    /* 0x01F78 */ SoundSource unk1F78[16];
```

Comment:
```suggestion
    /* 0x01F78 */ SoundSource soundSources[16];
```
Any reason to not name this now?